src/vma/dev/qp_mgr.h

```diff
@@ -215,6 +215,24 @@ class qp_mgr_eth : public qp_mgr
 			throw_vma_exception("failed creating qp");
 	};
 
+	static bool     is_mlx5_qp_supported(const ib_ctx_handler* ib_ctx) {
+		#define VMA_MLX5_MMAP_GET_WC_PAGES_CMD  2
+		#define VMA_MLX5_IB_MMAP_CMD_SHIFT      8
+
+		if (strstr(((ib_ctx_handler*)ib_ctx)->get_ibname(), "mlx5")) {
```

Comment:
https://github.com/Mellanox/libvma/blob/6416e3e9e0ecf39d1c96addbdc3008105a16d7f4/src/vma/dev/qp_mgr.h#L144

Reply:
I do not understand what that means,
I am planning to remove this function because no one is using it.